value/matrix.go

```diff
@@ -495,3 +495,52 @@ func (m *Matrix) dtranspose(c Context, v Vector) *Matrix {
 
 	return NewMatrix(shape, data)
 }
+
+// sel returns the selection of m according to v.
+// Like in APL, the selection applies to the final axis.
```

Comment:
the "Like in APL" is redundant. this whole program is mostly "like in APL".

Reply:
Done.